Clinical-Genomics/scout
# Replay managed variant - fix #1105

This PR adds skeletal managed variants - see #1105 for details.

A user can add variants to a managed list:
![Screenshot 2020-09-03 at 13 02 09](https://user-images.githubusercontent.com/758570/92107253-c5cc0f80-ede5-11ea-973b-6960fce72d78.png)

and see matches on the case pages:
![Screenshot 2020-09-03 at 13 03 35](https://user-images.githubusercontent.com/758570/92107373-f449ea80-ede5-11ea-8749-5612308a5113.png)

and variant page:
![Screenshot 2020-09-16 at 17 40 01](https://user-images.githubusercontent.com/758570/93360399-e794b000-f843-11ea-88b3-ec6cac6b9c89.png)

Loading a csv file should work:
![Screenshot 2020-09-21 at 11 10 09](https://user-images.githubusercontent.com/758570/93752968-b4fe0500-fbff-11ea-9e23-18314d116c1f.png)

Filters should work on the existing managed variants:
![Screenshot 2020-09-30 at 17 42 01](https://user-images.githubusercontent.com/758570/94708126-88a36080-0344-11eb-953d-3a6b582b0e27.png)

A complete version should: 
- [x] check matching variants on load, ensuring always load much like clinvar P/LP
- [x] show on the variant level
~~- [ ] ideally always show in clinical filter, though I think we may have to forgo that one for the time being~~
- [x] allow loading a larger list of managed variants, e.g. a csv or possibly vcf with variants
~~- [ ] be firmer on maintainer, institute and collaborator access to different operations on managed variants~~
~~- [ ] look a little crisper on the interface~~

**How to test**:
1. add a managed variant
1. modify it
1. remove it
1. load a csv. One is included in the demo dir `scout/demo/test_managed_variants.csv` - but hand-editing might uncover more issues.
1. filter on some criteria
1. pick a case with a variant in the managed_variant list (e.g. POT1 or what not): see managed variant on case and variant level.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
